elasticsearch-curator.spec

```diff
@@ -0,0 +1,58 @@
+########################################################################################
+
+%define package_name      curator
+
+########################################################################################
+
+Summary:        Utility for tending Elasticsearch indices 
+Name:           elasticsearch-%{package_name}
+Version:        4.0.6
+Release:        0%{?dist}
+License:        ASLv2.0
+Group:          Development/Libraries
+URL:            https://github.com/elastic/curator
+
+Source:         https://github.com/elastic/%{package_name}/archive/v%{version}.tar.gz
+
+BuildRequires:  python-devel python-setuptools
+
+BuildRoot:      %{_tmppath}/%{name}-%{version}-%{release}
```

Comment:
`BuildRoot` should be `%{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)` to avoid build conflicts with other users